src/Controller/AuthController.ts

```diff
@@ -63,8 +63,8 @@ export class AuthController extends BaseHttpController {
     }
 
     const verifyMFAResponse = await this.verifyMFA.execute({
-      email: <string> request.query.email,
-      token: <string> request.query.mfa_key,
+      email: request.query.email as string,
+      token: request.query.enteredTotp as string,
```

Comment:
hmm won't this break currently existing clients?

Reply:
Right, this should be released as Auth 2.0.0. Forgot to include the change to package.json.

Reply:
updated

Reply:
change in `package.json` are irrelevant to be honest on our "application projects" - they only matter on the `snjs` monorepo where we have packages.

on applications like this one every merge to `develop` gets published to DEV environment and every change merged to `main` branch gets published to PRODUCTION environment

Reply:
so how do we handle breaking changes like this?

Reply:
(discussion moved to the ticket)